app/views/root/_base_page.html.erb

```diff
@@ -20,6 +20,8 @@
   </div>
 </main>
 
-<% if local_assigns.fetch(:show_related_items, true) %>
-  <div id="related-items"></div>
+<% if local_assigns.fetch(:show_related_items, true) && @navigation_helpers %>
```

Comment:
We can avoid this conditional after: https://github.com/alphagov/frontend/pull/1033